operator/tests/central/basic-central/100-assert.yaml

```diff
@@ -1,18 +1,7 @@
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
-  name: stackrox-db
-spec:
-  accessModes:
-  - ReadWriteOnce
-  resources:
-    requests:
-      storage: 100Gi
----
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: stackrox-db
```

Comment:
Oh, wow, looks like we had a bug in the test!
Perhaps duplicate entries should be flagged by kuttl, let me file an upstream issue about this...

Reply:
FTR, https://github.com/kudobuilder/kuttl/issues/474